scipy/spatial/distance.py

```diff
@@ -1211,7 +1211,7 @@ def canberra(u, v, w=None):
     return d
 
 
-def jensenshannon(p, q, base=None):
+def jensenshannon(p, q, base=None, axis=0):
```

Comment:
```suggestion
def jensenshannon(p, q, base=None, *, axis=0, keepdim=False):
```

Reply:
I'm not sure what the `*` is for?

Reply:
This is to say that in order to use the parameters after it, you must call them explicitly `axis=0`, you cannot pass just `0`.

Reply:
I am not sure about the need for `keepdim`. @peterbell10 ?

If we add it, we will need to add `.. versionadded:: 1.7.0` here too.

Reply:
Thanks @tupui I didn't know that.

> If we add it, we will need to add .. versionadded:: 1.7.0 here too.
I added it.